### PR TITLE
stable/prometheus-postgres-exporter: allow for more options

### PR DESCRIPTION
**Pull request by jleloup**
#### Is this a new chart

No

#### What this PR does / why we need it:

The Prometheus postgres-exporter from https://github.com/wrouesnel/postgres_exporter does allow fore some options that could be nice to leverage in this chart:
* disabling monitoring queries on pg_settings: this is not needed when you are using this exporter for very specific queries.
* allow for setting different log levels for debugging purposes

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/prometheus-postgres-exporter/values.yaml [hunk 1]
```diff
@@ -75,8 +75,10 @@ config:
     database: ''
     sslmode: disable
   disableDefaultMetrics: false
+  disableSettingstMetrics: false
```
Comment (DerekHeldtWerle) [inline stable/prometheus-postgres-exporter/values.yaml]: Hey @jleloup,  is this supposed to be [disableSettingsMetrics](https://github.com/helm/charts/blob/b55555504e124c08535c02ffa2eed05dd17f6100/stable/prometheus-postgres-exporter/templates/deployment.yaml#L42) vs. `disableSettingstMetrics`? Seeing `disableSettingstMetrics` in the README table as well.

Comment (jleloup) [reply stable/prometheus-postgres-exporter/values.yaml]: This is definitely a typo. I'm gonna fix this

**Comment by jleloup** [comment]
@viglesiasce Hello, do you have an estimation of when you will have time to review this PR ? Also is there something I can do to help with that ?

**Comment by jleloup** [comment]
@prydonius @paulczar Would you have some time to review this PR ? Is there anything missing I should add ?

**Pull request closed without merging**